### ISSUE TITLE
Airdrop or Bluetooth does not work properly after wakeup from sleep

### ISSUE THREAD

**Issue by imraan-go** [xxxzc/xps15-9570-macos#76]
Airdrop stops working after a sleep/wake cycle. Works fine after a restart. Big Sur 11.1

**Comment by shasen1235** [comment]
Are you using Intel or Broadcom wireless card?
I use Intel AX200 and some user on itlwm repo are saying that Big Sur has some problem refreshing DHCP, even on M1 MacBook.
-Rolled back to 10.15 seems to solve this issue.
-Another thing to address is that 1.0 release IMO is more stable than 1.2 alpha.
-Also you might want to check the order of kext injection, which is quite important on OpenCore.
I found that
IntelBluetoothFirmware.kext
IntelBluetoothInjector.kext
AirportItlwm.kext
works the best for me.

**Comment by imraan-go** [comment]
I'm using DW1830 Broadcom wireless. the wifi is naively supported.

**Comment by imraan-go** [comment]
@xxxzc  just a heads up. any comments regarding the issue?

**Comment by adityahota** [comment]
I also have this issue. I am using a natively supported Broadcom card (so no BRCM* kexts needed) and everything was fine with 10.15 and 11.0.1. It is broken for me too on 11.1.

I'm going to upgrade to 11.2 today and let you know if that makes things better on my end.

**Comment by imraan-go** [comment]
> I also have this issue. I am using a natively supported Broadcom card (so no BRCM* kexts needed) and everything was fine with 10.15 and 11.0.1. It is broken for me too on 11.1.
> 
> I'm going to upgrade to 11.2 today and let you know if that makes things better on my end.

i'm using broadcom card too (DW 1830) but i think we still need brcm kexts. I'm on 11.2 already and this issue still persists. As far as i remember, I had this issue in catalina too.

**Comment by adityahota** [comment]
Yeah, upgrading to 11.2 didn't solve the problem for me either. Are you still able to use Bluetooth for non AirDrop/Continuity purposes (like headphones)?

I have the BCM94360NG card which is natively supported in macOS so I no longer use the kexts.

**Comment by imraan-go** [comment]
> Yeah, upgrading to 11.2 didn't solve the problem for me either. Are you still able to use Bluetooth for non AirDrop/Continuity purposes (like headphones)?
> 
> I have the BCM94360NG card which is natively supported in macOS so I no longer use the kexts.

I didn't know about this card. So everything used to work in Catalina for you?

**Comment by adityahota** [comment]
Yeah, everything works natively! Even instant hotspot and continuity clipboard.

**Comment by imraan-go** [comment]
> Yeah, everything works natively! Even instant hotspot and continuity clipboard.

I think your solution is here : https://www.tonymacx86.com/threads/bluetooth-stopped-working-after-updating-to-10-15-1-bcm943602cs.286706/page-5#post-2031968

**Comment by adityahota** [comment]
Unfortunately I tried those steps, and it looks like my card is not stuck in DFU mode. Thank you though!

**Comment by khoa-nv** [comment]
> Are you using Intel or Broadcom wireless card? I use Intel AX200 and some user on itlwm repo are saying that Big Sur has some problem refreshing DHCP, even on M1 MacBook. -Rolled back to 10.15 seems to solve this issue. -Another thing to address is that 1.0 release IMO is more stable than 1.2 alpha. -Also you might want to check the order of kext injection, which is quite important on OpenCore. I found that IntelBluetoothFirmware.kext IntelBluetoothInjector.kext AirportItlwm.kext works the best for me.

Hi, I'm on 13.5 and run Intel AX210. The wifi working perfect but Bluetooth not. Its show list of device but cannot pair. I see that the System report => Bluetooth => Discoverable is Off. Do you have any solution?
